gene_code_utils/dna_rna_tools_utils.py

```diff
@@ -0,0 +1,176 @@
+DNA = set ('ATGCatgc')
+RNA = set ('AUGCaugc')
+TRANSCRIPTION_TABLE = {
+    "a": "a", "A": "A",
+    "t": "u", "T": "U",
+    "u": "t", "U": "T",
+    "g": "g", "G": "G",
+    "c": "c", "C": "C"
+}
+
+
+def is_dna(seq: str) -> bool:
+    """
+    Check if a sequence is DNA.
+
+    Args:
+        seq (str): The input sequence to check.
+
+    Returns:
+        bool: True if the sequence is DNA, False otherwise.
+    """
+    unique_chars = set(seq)
+    return unique_chars <= DNA
+
+
+def is_rna(seq: str) -> bool:
+    """
+    Check if a sequence is RNA.
+
+    Args:
+        seq (str): The input sequence to check.
+
+    Returns:
+        bool: True if the sequence is RNA, False otherwise.
+    """
+    unique_chars = set(seq)
+    return unique_chars <= RNA
+
+
+def type_rna_or_dna(seqs: List[str]) -> str:
+    """
+    Determine the type of RNA or DNA from a list of sequences.
+
+    Args:
+        seqs (List[str]): List of sequences to determine their type.
+
+    Returns:
+        str: 'DNA' if all sequences are DNA, 'RNA' if all sequences are RNA,
+            'MIXED' if there are both DNA and RNA sequences, or raises a ValueError for unsupported sequences.
+    """
+    counter_dna = 0
+    counter_rna = 0
+    ambigiuos = 0
+    for i in seqs:
+      if is_dna(i) and is_rna(i):
```

Comment:
Не хватает отступов.
```suggestion
        if is_dna(i) and is_rna(i):
```